tests/integration/testdata/package/aws-serverless-inline.yaml

```diff
@@ -12,9 +12,3 @@ Resources:
           return {'body': 'Hello World 2!', 'statusCode': 200}
       Handler: index.handler
       Runtime: python3.7
-      Events:
-        Api:
```

Comment:
Also why are we removing it?

Reply:
Not need, and would require more permissions for our integration tests to work.